setup.py

```diff
@@ -1,18 +1,20 @@
-# vim:fileencoding=utf-8:noet
-
+# vim:fileencoding=utf-8:noet:tabstop=4:softtabstop=4:shiftwidth=4:expandtab:
+import pathlib
 from setuptools import setup
 
 setup(
     name             = 'powerline-kubernetes',
     description      = 'A Powerline segment to show Kubernetes context',
+    long_description = (pathlib.Path(__file__).parent / "README.md").read_text(),
+    long_description_content_type="text/markdown",
     version          = '1.0.2',
     keywords         = 'powerline kubernetes context',
     license          = 'MIT',
     author           = 'Vincent De Smet',
```

Comment:
do you want to be added here?

Reply:
That would be lovely, thank you.

I'll fix both issues, squash the commits and revert back.